Build fail

```
$ cmake .. -DOKON_WITH_CLI=ON -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=../../install                               (master) 
-- Configuring done
You have changed variables that require your cache to be deleted.
Configure will be re-run and you may have to reset some variables.
The following variables have changed:
CMAKE_CXX_COMPILER= g++

-- The C compiler identification is GNU 10.1.0
-- The CXX compiler identification is GNU 10.1.0
-- Check for working C compiler: /usr/lib/ccache/bin/cc
-- Check for working C compiler: /usr/lib/ccache/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/lib/ccache/bin/g++
-- Check for working CXX compiler: /usr/lib/ccache/bin/g++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/okon/okon/build
```

```
[user:/tmp/okon/okon/build]$ make install                                                                                                            (master) 
Scanning dependencies of target okon
[ 16%] Building CXX object lib/CMakeFiles/okon.dir/btree_node.cpp.o
In file included from /tmp/okon/okon/lib/btree_node.hpp:3,
                 from /tmp/okon/okon/lib/btree_node.cpp:1:
/tmp/okon/okon/lib/sha1_utils.hpp:41:42: ошибка: «string_view» не является элементом «std»
   41 | inline sha1_t string_sha1_to_binary(std::string_view sha1_text)
      |                                          ^~~~~~~~~~~
/tmp/okon/okon/lib/sha1_utils.hpp:10:1: замечание: «std::string_view» is defined in header «<string_view>»; did you forget to «#include <string_view>»?
    9 | #include <vcl/vectorclass.h>
  +++ |+#include <string_view>
   10 | 
/tmp/okon/okon/lib/sha1_utils.hpp:42:1: ошибка: expected «,» or «;» before «{» token
   42 | {
      | ^
/tmp/okon/okon/lib/sha1_utils.hpp:88:13: ошибка: «string» в пространстве имен «std» не именует тип
   88 | inline std::string binary_sha1_to_string(const sha1_t& sha1)
      |             ^~~~~~
/tmp/okon/okon/lib/sha1_utils.hpp:10:1: замечание: «std::string» is defined in header «<string>»; did you forget to «#include <string>»?
    9 | #include <vcl/vectorclass.h>
  +++ |+#include <string>
   10 | 
In file included from /tmp/okon/okon/lib/btree_node.cpp:1:
/tmp/okon/okon/lib/btree_node.hpp:15:49: ошибка: «numeric_limits» не является элементом «std»
   15 |   static constexpr auto k_unused_pointer = std::numeric_limits<pointer_t>::max();
      |                                                 ^~~~~~~~~~~~~~
/tmp/okon/okon/lib/btree_node.hpp:15:73: ошибка: expected primary-expression before «>» token
   15 |   static constexpr auto k_unused_pointer = std::numeric_limits<pointer_t>::max();
      |                                                                         ^
/tmp/okon/okon/lib/btree_node.hpp:15:76: ошибка: «::max» не был декларирован
   15 |   static constexpr auto k_unused_pointer = std::numeric_limits<pointer_t>::max();
      |                                                                            ^~~
/tmp/okon/okon/lib/btree_node.hpp:15:76: замечание: предлагаемые варианты:
In file included from /usr/include/c++/10.1.0/array:40,
                 from /tmp/okon/okon/lib/sha1_utils.hpp:3,
                 from /tmp/okon/okon/lib/btree_node.hpp:3,
                 from /tmp/okon/okon/lib/btree_node.cpp:1:
/usr/include/c++/10.1.0/bits/stl_algobase.h:300:5: замечание:   «std::max»
  300 |     max(const _Tp& __a, const _Tp& __b, _Compare __comp)
      |     ^~~
In file included from /tmp/okon/okon/3rdparty/vcl/vectorclass.h:71,
                 from /tmp/okon/okon/lib/sha1_utils.hpp:9,
                 from /tmp/okon/okon/lib/btree_node.hpp:3,
                 from /tmp/okon/okon/lib/btree_node.cpp:1:
/tmp/okon/okon/3rdparty/vcl/vectori512se.h:1590:23: замечание:   «vcl::max»
 1590 | static inline Vec32us max(Vec32us const a, Vec32us const b) {
      |                       ^~~
In file included from /tmp/okon/okon/lib/btree_node.cpp:1:
/tmp/okon/okon/lib/btree_node.hpp:37:46: ошибка: cannot convert «<brace-enclosed initializer list>» to «okon::btree_node::pointer_t» {aka «unsigned int»} in initialization
   37 |   pointer_t parent_pointer{ k_unused_pointer };
      |                                              ^
make[2]: *** [lib/CMakeFiles/okon.dir/build.make:83: lib/CMakeFiles/okon.dir/btree_node.cpp.o] Ошибка 1
make[1]: *** [CMakeFiles/Makefile2:114: lib/CMakeFiles/okon.dir/all] Ошибка 2
make: *** [Makefile:150: all] Ошибка 2
```

--- a/lib/btree_node.hpp
+++ b/lib/btree_node.hpp
@@ -3,6 +3,7 @@
 #include "sha1_utils.hpp"
 
 #include <cstdint>
+#include <limits>
 #include <vector>
 
 namespace okon {

--- a/lib/sha1_utils.hpp
+++ b/lib/sha1_utils.hpp
@@ -4,6 +4,8 @@
 #include <cassert>
 #include <cstdint>
 #include <cstring>
+#include <string>
+#include <string_view>
 
 #define VCL_NAMESPACE vcl
 #include <vcl/vectorclass.h>
@@ -62,7 +64,7 @@ inline sha1_t simd_string_sha1_to_binary(const void* text)
   v8.load(text);
 
   const auto is_alpha_mask = (v8 & 0b01000000u) != 0u;
-  
+
   v8 &= vcl::Vec64uc{ 0x0fu };
   v8 = if_add(is_alpha_mask, v8, vcl::Vec64uc{ 9u });
 

